registry/templates/donor/detail.html

```diff
@@ -89,6 +89,8 @@ <h2>Poznámky</h2>
 </div>
 <h2>Historie importů</h2>
 
+<a href="{{ url_for('batch.import_data', rodne_cislo=overview.rodne_cislo) }}" class="btn btn-primary">Připravit manuální import</a>
```

Comment:
Uvažuji nad tou formulací _Připravit manuální import_ zda tam nenechat jen _Manuální import,_ nebo něco podobného?

Reply:
Tím "připravit" jsem se snažil říci, že to jen nachystá vstup a zbytek importu už probíhá klasicky. Ale není to nic zásadního, takže to klidně změním.